pkg/blob/nodeserver.go

```diff
@@ -409,6 +409,7 @@ func (d *Driver) NodeStageVolume(ctx context.Context, req *csi.NodeStageVolumeRe
 	}
 
 	// Get mountOptions that the volume will be formatted and mounted with
+	mountFlags = util.JoinMountOptions(mountFlags, []string{fmt.Sprintf("--telemetry=azpartner-aks/%s", d.Version)})
```

Comment:
since blobfuse v1 does not support this option, it would break, pls add telemetry into blobfuse2 example config: https://github.com/kubernetes-sigs/blob-csi-driver/blob/master/deploy/example/storageclass-blobfuse2.yaml

Reply:
This telemetry is to track usage of azure blob storage through AKS, we can't add it in example config since there will be high chances that user can avoid it. I have updated mountFlag only if protocol is fuse